var/spack/repos/builtin/packages/xsdk/package.py

```diff
@@ -64,17 +64,17 @@ class Xsdk(BundlePackage):
     depends_on('superlu-dist@5.2.2', when='@0.3.0')
     depends_on('superlu-dist@xsdk-0.2.0', when='@xsdk-0.2.0')
 
-    depends_on('trilinos@develop+hypre+superlu-dist+metis+hdf5~mumps+boost~suite-sparse+tpetra+nox+ifpack2+zoltan2+amesos2~exodus~dtk+intrepid2+shards+stratimikos gotype=int cxxstd=14',
+    depends_on('trilinos@develop+hypre+superlu-dist+hdf5~mumps+boost~suite-sparse+tpetra+nox+ifpack2+zoltan2+amesos2~exodus~dtk+intrepid2+shards+stratimikos gotype=int cxxstd=14',
```

Comment:
looks like `+zoltan` is an internal dependency for `+zoltan2`?
```
spack spec xsdk:

1. "+zoltan2" conflicts with "trilinos~zoltan"
```
I'm trying out the following fix 

https://github.com/spack/spack/commit/224581142602452d1cf8f6cedf7a019d13832f85

https://gitlab.com/xsdk-project/spack-xsdk/-/pipelines/337795803

Reply:
Ooooh, I think I misread the trilinos dependencies here. It's got a *test* required dependencies but not a *build* dep. I will revert this. Thanks!

Reply:
Wait sorry, Zoltan2 requires Zoltan2Core which requires Zoltan. It looks like your suggested fix is right. Are you concretizing with the original or clingo? I can cherry-pick your commit onto this branch.

Reply:
> Are you concretizing with the original or clingo

I have no idea what they are. I guess I'm using whatever spack defaults to..

> I can cherry-pick your commit onto this branch

Sure.